css/main.css

```diff
@@ -0,0 +1,188 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  max-width: 100%;
+  overflow-x: hidden;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
- [ ] Kindly use the required `fonts` for this project. The required fonts are `Cocogoose` and `Lato`. Kindly implement this change. 👍

Reply:
Hi, @Gambit142 I must commend you. You are very thorough in reviewing. Thank you.